languages/MFS.groovy

```diff
@@ -63,6 +73,8 @@ sortedList.each { buildFile ->
 			props.error = "true"
 			buildUtils.updateBuildResult(errorMsg:errorMsg,logs:["${member}.log":logFile])
 		}
+		
+		}
```

Comment:
For better readability, can you change the indentation of the block between L65 and L75?
Thank you!

Reply:
@M-DLB Applied pretty print.